src/compute-types/src/dataflows.rs

```diff
@@ -72,6 +73,8 @@ pub struct DataflowDescription<P, S: 'static = (), T = mz_repr::Timestamp> {
     pub refresh_schedule: Option<RefreshSchedule>,
     /// Human readable name
     pub debug_name: String,
+    /// The timeline of the dataflow.
+    pub timeline: Option<Timeline>,
```

Comment:
Yes changing this to `is_timeline_epochms` seems to be more appropriate.

Reply:
Could also change it to `can_expire` or somesuch. This would come in handy with my imagined solution for the `REFRESH` MV issue, which involves checking whether the dataflow transitively depends on a `REFRESH` MV and disabling expiration if it does. The checking needs to happen in the coordinator, so the disabling needs to be communicated in the `DataflowDescription`. There is no reason to have two bools if one is sufficient.

Reply:
Yes, that is exactly what I'm working on next.

Reply:
Updated.